packages/color-area/test/color-area.test.ts

```diff
@@ -547,6 +547,8 @@ describe('ColorArea', () => {
         );
 
         await elementUpdated(el);
+        const Xvalue = Number(Number(el.inputX.value).toFixed(2));
+        const Yvalue = Number(Number(el.inputY.value).toFixed(2));
```

Comment:
Can we fast forward unrelated changes into a standalone PR that we can merge first?

Reply:
These are just some tests for color-area that I added to increase the code coverage. Other than this I didn't make any changes in the color-area component!

Reply:
@blunteshwar I think Westbrook is right! lets not pollute this PR with color-area changes.